rclpy/src/rclpy/_rclpy.c

```diff
@@ -2520,11 +2520,14 @@ static PyMethodDef rclpy_methods[] = {
   {NULL, NULL, 0, NULL}  /* sentinel */
 };
 
+PyDoc_STRVAR(rclpy__doc__,
+  "RCLPY: module for ROS Python client library.");
```

Comment:
The name `rclpy` and the fact that it is a module are already clear by the module name itself. Therefore I would suggest to boil it down to:

> ROS 2 Python client library.